### ISSUE TITLE
[NPC] [Boss] [Karazhan][Curator] - Paladin Threat Problem/Boss Dying

### ISSUE THREAD

**Issue by CostinRaz** [Atlantiss/NetherwingBugtracker#2955]
**Current behaviour**:

First Issue: Paladins seems to generate very little threat on the boss, low enough that a healer can easily steal aggro on the fight very early on despite the paladin using Wing+AS+Judge+Consecration. Another tank will also be able to easily steal aggro. Issue might be related to how paladin threat works via righteous fury + Curator's dmg reduction.

Second Issue: Boss Simply died for us after first evocate when he was around 50%. 

**Expected behaviour**: 

First Issue: Paladin should be able to generate enough burst aggro at start to easily hold on to the boss at least against healers, here's an example video.

https://www.youtube.com/watch?v=KJN5laBkInI

Second Issue: Boss should enter a second normal phase.

**Server Revision**: 2351

**Comment by tcberk12** [comment]
The second issue was resolved in rev 2350, per discord changelog.

[2350][Core] Fixed an issue which could sometimes cause the melee damage to be negative, causing an overflow and killing the target instantly (e.g. The Curator)

@CostinRaz Are you sure the second issue occured on rev 2351?

**Comment by CostinRaz** [comment]
@tcberk12 Yes it was after the introduction of 2351 on the server.

**Comment by tcberk12** [comment]
Ok, and about how long ago did you experience the issue? time wise

**Comment by Easyfraggy** [comment]
Leaving this open for 1 more day - if no further evidence of a bug I will close this issue.

**Comment by Easyfraggy** [comment]
There's no Astral Armor on neither of those videos.

**Comment by CostinRaz** [comment]
When was it removed exactly? My bad on the links I fixed that. 

If you mean the idea that curator has a dmg reduction buff that results in raids not being to do any sort of dmg on him before evocate then yes I have, if you mean the idea he has a big shinny block buff around himself then no. Because it was never there on retail as far as I remember.

I searched around: The oldest kill I could find was this: https://www.youtube.com/watch?v=2whzzWmpnvU

January the 30th 2007, two weeks after the expansion came out, he has no visual reference to it.
